app/src/main/java/mobile/programming/musicapp/MainActivity.kt

```diff
@@ -2,7 +2,7 @@ package mobile.programming.musicapp
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import mobile.programming.musicapp.R
+//import mobile.programming.musicapp.R
```

Comment:
Commented out the unused imports in case they are needed for implementing later.

Reply:
Avoid committing commented out code to version control.  This is a no-no, as version control offers better alternatives.
- You're in your own branch, so you can do what you want to do without impacting master.  This gives you a license to experiment.  Instead of commenting out code, delete it.
- You can always go back in history to see what used to be there.